org.kde.dragonplayer.json

```diff
@@ -0,0 +1,39 @@
+{
+  "app-id": "org.kde.dragonplayer",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.15-22.08",
+  "sdk": "org.kde.Sdk",
+  "command": "dragon",
+  "rename-icon": "dragonplayer",
+  "finish-args": [
+    "--share=ipc",
+    "--share=network",
+    "--socket=wayland",
+    "--socket=fallback-x11",
+    "--socket=pulseaudio",
+    "--device=dri",
+    "--filesystem=home",
+    "--filesystem=/run/media",
+    "--filesystem=/media",
```

Comment:
does it not use the portal?

Reply:
Good point. Will give it a try

Reply:
Perhaps it does use file portals but portals are too limited for media apps (loading subtitles to a movie, playlists. Other published apps don't use [them](https://github.com/flathub/org.kde.elisa/blob/master/org.kde.elisa.json#L13).

There is even nasty portal [bug](https://github.com/flatpak/xdg-desktop-portal/issues/689) when it doesn't close once opened files thus prevent unmounting removable media.

Reply:
Reduced the default permissions.